scripts/release/oci_chart_info.py

```diff
@@ -0,0 +1,24 @@
+import json
+import os
+import sys
+from dataclasses import asdict
+
+from lib.base_logger import logger
+from scripts.release.build.build_info import load_build_info
+
+
+def main():
+    build_scenario = os.environ.get("BUILD_SCENARIO")
```

Comment:
Let's not hide the dependency on "BUILD_SCENARIO" env var here and pass it directly as cmd flag.